docs/developers/developing_iree/profiling_with_tracy.md

```diff
@@ -1,15 +1,16 @@
 # Profiling with Tracy
 
 [Tracy](https://github.com/wolfpld/tracy) is a profiler that puts together in a
-single view:
-
-*   Both CPU and GPU profiling.
-*   Both sampling and instrumentation.
-*   Both specifics of our own process, and whole-system profiling a la
-    "systrace".
-
-Since Tracy relies on instrumentation, it requires IREE binaries to be built
-with a special flag to enable it.
+single view both instrumentation and system profiling (sampling, systrace). It's
+key to understand the nuance here.
+* *Instrumentation* is code built into the process being profiled, collecting
+  timestamps at the start and end of "zones". Once it's enabled at build time,
+  it typically just works &emdash; it is a part of our application logic just
```

Comment:
Looks like this `&emdash;` isn't showing up in GitHub's markdown render?

Reply:
because i had it wrong, it's `&mdash;` - fixed.